### ISSUE TITLE
Pokemon Platinum 

### ISSUE THREAD

**Issue by fennectech** [TuxSH/TWLSaveTool#13]
it says "unsupported save type"  but it doesnt say so in your wiki instructions thingy

**Comment by TuxSH** [comment]
This game works for me as well as many other people, the issue is probably on your end.

**Comment by gotzathe** [comment]
I'm also having issues with this game. Other titles like Pokemon SoulSilver are working fine.

**Comment by SyedDawoud** [comment]
Pokemon Platinum works fine. I only face trouble with Soulsilver as TWLsavetool freezes on launch.

**Comment by fennectech** [comment]
Ive used soul silver and diamond

**Comment by SyedDawoud** [comment]
How does soulsilver works? on my two 3ds', TWLsavetool freezes on version line with soulsilver.

**Comment by NataliePequeno** [comment]
I don't even get any worded error message.
If I use Pokemon White 2, it works fine, but if I use Platinum, I get An exception occurred, Processor: arm11, exception type: data abort and a load of numbers :/ Is there something wrong with my copy?

**Comment by Supermariotyt** [comment]
Platiunum must have a problem with it. I Used super mario 64ds and it worked, but i tried pkm platinum and it failed

**Comment by map1e133** [comment]
I'm having the same issue, only on Pokémon Platinum. Black2, Soulsilver and White works just fine.

**Comment by FM1337** [comment]
@NataliePequeno @Supermariotyt  see issue #20

**Comment by fennectech** [comment]
The issue is bootleg copy’s of the game confuse twlsavetool

**Comment by Supermariotyt** [comment]
@fennectech Yes, that could be possible. My game was from ebay and it came from china. It freezes a ton.

**Comment by Lollipop25** [comment]
Hello i update this thread because i have a solution for fix that and import/export pkm platinium's save i used checkpoint and works perfectly .

[Here link](https://github.com/FlagBrew/Checkpoint/releases)

maybe works with pokemon heartgold/soulsilver.
